<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GridZoneGeneratorDialogBase</class>
 <widget class="QDialog" name="GridZoneGeneratorDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grid Zone Generator</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="createShapeBox">
     <property name="title">
      <string>Create shapefile layer? (Instead of temporary layer)</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Destination File</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="saveEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveButton">
          <property name="text">
           <string>Save File</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Desired Grid Zone Scale: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="stopScaleCombo">
       <item>
        <property name="text">
         <string>1000k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>500k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>250k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>50k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>25k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1k</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>CRS: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="crsLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="crsButton">
       <property name="text">
        <string>Select CRS</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Reference Data</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Base Scale: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="scaleCombo">
          <item>
           <property name="text">
            <string>1000k</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>500k</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>250k</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>100k</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>50k</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>25k</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10k</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5k</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2k</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1k</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Index Frame Selection</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QRadioButton" name="mirRadioButton">
             <property name="text">
              <string>MIR</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="mirLineEdit">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QRadioButton" name="miRadioButton">
             <property name="text">
              <string>MI</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="miLineEdit">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QRadioButton" name="indexRadioButton">
             <property name="text">
              <string>INDEX</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="indexLineEdit">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>GridZoneGeneratorDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>GridZoneGeneratorDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
